__tests__/app.test.js

```diff
@@ -360,6 +359,7 @@ describe("/api/articles(topic)", () => {
         .expect(200)
         .then((response) => {
           expect(response.body).toBeInstanceOf(Array);
+
```

Comment:
It's good that you are checking for the topic in each returned object, however only the key. They could actually be empty?